<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(145, 204, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FINANCEIRO  -  INSTITUTO BEIJA-FLOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGerarPdf">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>800</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 130, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>GERAR PDF</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblColabPlanilha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>691</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>NOME</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CARGO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SEGMENTO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VALOR</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tblConsultarGastos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>451</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>Descrição</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lblNovoSaldo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>740</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 129, 0);</string>
    </property>
    <property name="text">
     <string>NOVO SALDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSaida">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>685</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(220, 0, 0);</string>
    </property>
    <property name="text">
     <string>SAÍDA PREVISTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSaldo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(221, 147, 0);</string>
    </property>
    <property name="text">
     <string>SALDO ATUAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>720</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SALDO PÓS PAGAMENTOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>600</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SALDO ATUAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>660</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SAÍDA PREVISTA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
